bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -218,9 +219,9 @@ private BigtableOptions(
     Preconditions.checkArgument(
         !Strings.isNullOrEmpty(projectId), "ProjectId must not be empty or null.");
     Preconditions.checkArgument(
-        !Strings.isNullOrEmpty(zone), "Zone must not be empty or null.");
+        !Strings.isNullOrEmpty(zoneId), "Zone must not be empty or null.");
```

Comment:
Does it make more sense to have "Zone ID must not be empty or null."
and 
"Cluster ID must not be empty or null."